.github/workflows/seed-data.yml

```diff
@@ -55,7 +55,11 @@ jobs:
           max_attempts: 3
           command: |
             cd ./opencrvs-core
-            yarn seed:prod
+            if [[ ${{ github.event.inputs.environment }} == 'production' ]]; then
+                yarn seed:prod
+            else
+                yarn seed:dev
+            fi
```

Comment:
We don't really need this as the only difference between these two commands is that the dev one activates the users by default. But there is a env variable to control whether we want to activate users or not in the github environment